platform/lib/pipeline/markdownDocument.js

```diff
@@ -213,9 +221,21 @@ class MarkdownDocument {
    */
   save(path) {
     const frontmatter = `---\n${yaml.safeDump(this._frontmatter, {'skipInvalid': true})}---\n\n`;
+    let content = frontmatter + this._contents;
+
+    if (this._importPath) {
+      const IMPORTED_TEXT = '<!-- \n' +
+      'This file is imported from https://github.com/ampproject/amphtml/blob/master/' + this.importPath +
```

Comment:
Also here use template strings

Reply:
also changed, makes it even more readable, thanks